server/ce2e/plugin_test.go

```diff
@@ -379,11 +379,11 @@ func TestSelectiveSync(t *testing.T) {
 
 	conn, err := mattermost.PostgresConnection(context.Background())
 	require.NoError(t, err)
+	defer conn.Close()
 
 	// Mark user as synthetic
-	_, err = conn.Exec("UPDATE Users SET RemoteId = 'syntetic-user' WHERE Id = $1", synthetic.Id)
+	_, err = conn.Exec("UPDATE Users SET RemoteId = (SELECT remoteId FROM remoteclusters) WHERE Username = 'msteams_synthetic'")
```

Comment:
Let's constrain `(SELECT remoteId FROM remoteclusters)` to only returning the `remoteId` for this plugin, otherwise this could return multiple results and fail.

Reply:
As it is part of the plugin test, my expectation is there is nothing else there, I can add it, but it shouldn't be a problem.